File: velolabs_repos/lattis_android/app/src/main/java/com/lattis/ellipse/data/network/model/response/card/SetUpIntentDataResponse.java
package com.lattis.ellipse.data.network.model.response.card;

import com.google.gson.annotations.SerializedName;
import com.lattis.ellipse.data.network.model.response.AbstractApiResponse;
import com.lattis.ellipse.data.network.model.response.bike.FindBikeDataResponse;

import org.json.JSONException;
import org.json.JSONObject;

import java.util.List;

public class SetUpIntentDataResponse {

    @SerializedName("id")
    private String id;

    @SerializedName("object")
    private String object;

    @SerializedName("application")
    private String application;

    @SerializedName("cancellation_reason")
    private String cancellation_reason;

    @SerializedName("client_secret")
    private String client_secret;

    @SerializedName("created")
    private long created;

    @SerializedName("customer")
    private String customer;

    @SerializedName("description")
    private String description;

    @SerializedName("last_setup_error")
    private String last_setup_error;

    @SerializedName("livemode")
    private boolean livemode;

    @SerializedName("next_action")
    private String next_action;

    @SerializedName("on_behalf_of")
    private String on_behalf_of;

    @SerializedName("payment_method")
    private String payment_method;

    @SerializedName("status")
    private String status;

    @SerializedName("usage")
    private String usage;

    @SerializedName("payment_method_types")
    private List<String> payment_method_types;


    public String getId() {
        return id;
    }

    public String getObject() {
        return object;
    }

    public String getApplication() {
        return application;
    }

    public String getCancellation_reason() {
        return cancellation_reason;
    }

    public String getClient_secret() {
        return client_secret;
    }

    public long getCreated() {
        return created;
    }

    public String getCustomer() {
        return customer;
    }

    public String getDescription() {
        return description;
    }

    public String getLast_setup_error() {
        return last_setup_error;
    }

    public boolean isLivemode() {
        return livemode;
    }

    public String getNext_action() {
        return next_action;
    }

    public String getOn_behalf_of() {
        return on_behalf_of;
    }

    public String getPayment_method() {
        return payment_method;
    }

    public String getStatus() {
        return status;
    }

    public String getUsage() {
        return usage;
    }

    public List<String> getPayment_method_types() {
        return payment_method_types;
    }

}

File: velolabs_repos/lattis_android_v2/data/src/main/java/com/lattis/data/entity/response/card/SetUpIntentDataResponse.kt
package com.lattis.data.entity.response.card

import com.google.gson.annotations.SerializedName

class SetUpIntentDataResponse {
    @SerializedName("id")
    val id: String? = null

    @SerializedName("object")
    val `object`: String? = null

    @SerializedName("application")
    val application: String? = null

    @SerializedName("cancellation_reason")
    val cancellation_reason: String? = null

    @SerializedName("client_secret")
    val client_secret: String? = null

    @SerializedName("created")
    val created: Long = 0

    @SerializedName("customer")
    val customer: String? = null

    @SerializedName("description")
    val description: String? = null

    @SerializedName("last_setup_error")
    val last_setup_error: String? = null

    @SerializedName("livemode")
    val isLivemode = false

    @SerializedName("next_action")
    val next_action: String? = null

    @SerializedName("on_behalf_of")
    val on_behalf_of: String? = null

    @SerializedName("payment_method")
    val payment_method: String? = null

    @SerializedName("status")
    val status: String? = null

    @SerializedName("usage")
    val usage: String? = null

    @SerializedName("payment_method_types")
    val payment_method_types: List<String>? = null

}